resources/modules/bans.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * MyBB 1.8 Merge System
+ * Copyright 2014 MyBB Group, All Rights Reserved
+ *
+ * Author: Brad Veryard (https://github.com/veryard)
+ * Website: http://www.mybb.com
+ * License: http://www.mybb.com/download/merge-system/license/
+ */
+
+abstract class Converted_Module_Bans extends Converter_Module
```

Comment:
This should be `Converter_Module_Bans` I believe:

```suggestion
abstract class Converter_Module_Bans extends Converter_Module
```

Reply:
Whoops! Had it named `Converted` in both cases, renamed for consistency.